package.json

```diff
@@ -13,6 +13,7 @@
     "babel-polyfill": "^6.26.0",
     "css-loader": "^3.1.0",
     "express": "^4.17.1",
+    "idempotent-babel-polyfill": "^7.4.4",
```

Comment:
@Alihaider168 are we using this library ? If not lets uninstall/remove it.

Reply:
just for testing added. im going to uninstall it.